spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -415,6 +421,36 @@ void ClientDLL::ResetAfterframesQueue()
 
 void ClientDLL::OnFrame()
 {
+#if !defined(OE)
+	extern int GetEntityCount();
+	extern CBaseEntity* GetEntity(int);
+
+	if (y_spt_piwsave.GetString()[0] != '\0')
+	{
+		int count = GetEntityCount();
+		for (int i = 0; i < count; ++i)
+		{
+			auto ent = GetEntity(i);
+			if (!ent)
+				continue;
+
+			auto phys = ent->VPhysicsGetObject();
```

Comment:
It’s not a very good idea to use these functions directly because their addresses change for pretty much every different game version, and the worst part is that there’s no failsafe: if you load this build of SPT on an incompatible game version and try using the CVar the game will either just crash or do something completely unpredictable.